src/content/community/language-resources/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Language resources
+description: Non-english resources to learn about Ethereum
+sidebar: true
+lang: en
+---
+
+# Language resources {#language-resources}
+
+The Ethereum community is global and comprised of millions of non-English speakers.
+
+Our aim is to provide educational content in all languages and help overcome the language barriers that make onboarding people from all over the world to Ethereum a challenge.
+
+If you prefer reading in your native language or know someone who doesn’t speak English, you can find a list of useful non-English resources below.
+
+Know of an educational resource in your language? [Open an issue](https://github.com/ethereum/ethereum-org-website/issues/new/choose) to add it to the list!
+
+If you are bilingual and want to help us reach more people, you can also get involved with the [ethereum.org Translation Program](https://ethereum.org/en/contributing/translation-program/#translation-program) and help us translate the website.
+
+Hundreds of thousands of Ethereum enthusiasts gather in these online forums to share news, talk about recent developments, debate technical issues, and imagine the future.
+
+## Community resources {#community}
+
+### Brazilian Portuguese {#br-pt}
+
+**News**
+
+- [BeInCrypto](http://www.beincrypto.com.br) - cryptocurrency news and articles, including a list of exchanges, available in Brazil
+- [Cointelegraph](http://cointelegraph.com.br/category/analysis) - Brazilian version of Cointelegraph, a major cryptocurrency news outlet
+- [Seudinheiro](http://www.seudinheiro.com/criptomoedas/) - cryptocurrency news and reports
+- [~~Investing.com](http://br.investing.com/news/cryptocurrency-news/) - Brazilian version of Investing.com, regularly covering cryptocurrency news~~
+- [~~InfoMoney](http://www.infomoney.com.br/tudo-sobre/criptomoedas/) - portal for inverstors with a section on cryptocurrency news and articles~~
+- [~~Money Times](http://www.moneytimes.com.br/cryptotimes/) - cryptocurrency news and market tracker~~
+- [~~Exame](http://www.exame.com/noticias-sobre/criptomoedas/) - investor education and news, with a ‘Cryptocurrency news’ subpage~~
```

Comment:
@minimalsm I guess we should remove the above 4 lines, right?

Reply:
Done!